### DISCUSSION TITLE
Setting up crabada.py

### DISCUSSION THREAD

**Discussion by codingfreek1** [coccoinomane/crabada.py#72] [Q&A] [answered]
Hello all,

I'm new to this project and wanted to try and test how the bot works. I have a windows machine with python installed.
Is there a quick step by step tutorial on the step up process?

Thanks in advance for your help.

**Comment by mertf** [comment]
Hi @codingfreek1,

The Readme file is very self-explanatory, and you shouldn't have any problems setting the bot up if you can follow a basic tutorial. However, everyone here is very helpful. You should get answers to your questions wherever you get stuck.

**Reply by codingfreek1** [reply]
My question is ..how to run the bot on windows? Could you help me?

**Comment by coccoinomane** [comment, marked as answer]
Hi @codingfreek1 ,

Have you tried following the [Quick start](https://github.com/coccoinomane/crabada.py#quick-start) section of the README?

Put in the initial effort and then hopefully we can help you 🙂

Cheers,
Cocco

**Reply by codingfreek1** [reply]
Ok sure. Thanks

**Reply by isidororeyes** [reply]
quick one. 
The USER_1_PRIVATE_KEY="needed to interact with the Crabada smart contract"
Is this Hex too? Or plaintext? 
Thanks

**Reply by coccoinomane** [reply]
@isidororeyes  The private key is the one you get when you export the account from MetaMask. It is a hex address, from which you need to remove the initial `0x`.

**Reply by isidororeyes** [reply]
Thanks @coccoinomane for your reply! Took me a while to get onboarded on the project but now I'm fully up to speed!! Also added @EugeneTeu TustoAvax conversion.

My plan is to work on a decision/strategy modeling! 
Will share when done!!
Thanks

**Reply by coccoinomane** [reply]
Great @isidororeyes , I am glad you managed to run the bot 💪

Good luck with your strategizing... and consider donating a few % to the project using the `DONATE` parameter in .env 😉

Cheers,
Cocco
